### ISSUE TITLE
FLOSS bug (modules not found) in Windows XP VM

### ISSUE THREAD

**Issue by stack-smasher** [fireeye/flare-floss#273]
I encountered a weird error when running the floss standalone executable in windows xp vm:

```
C:\>floss32 file.exe > output.txt
WARNING:vtrace.platforms.win32:LoadLibrary C:\DOCUME~1\user\LOCALS~1\Temp_MEI40
~1\vtrace\platforms\windll\i386\symsrv.dll: [Error 126] The specified module cou
ld not be found
WARNING:vtrace.platforms.win32:LoadLibrary C:\DOCUME~1\user\LOCALS~1\Temp_MEI40
~1\vtrace\platforms\windll\i386\dbghelp.dll: [Error 126] The specified module co
uld not be found
Traceback (most recent call last):
File "floss\main.py", line 887, in <module>
File "floss\main.py", line 850, in main
File "floss\main.py", line 72, in decode_strings
File "floss\string_decoder.py", line 140, in emulate_decoding_routine
File "floss\decoding_manager.py", line 157, in emulate_function
File "site-packages\viv_utils\emulator_drivers.py", line 338, in runToVa
File "site-packages\viv_utils\emulator_drivers.py", line 298, in stepi
File "site-packages\viv_utils\emulator_drivers.py", line 287, in step
File "site-packages\envi\archs\i386\emu.py", line 198, in executeOpcode
File "site-packages\envi\archs\i386\emu.py", line 1004, in i_movsd
File "site-packages\vivisect\impemu\emulator.py", line 541, in writeMemory
File "site-packages\envi\memory.py", line 452, in writeMemory
KeyboardInterrupt
Failed to execute script main
```
------
The "output.txt" file was still created and contained output results so I'm not sure what the error shown above signified.

**Comment by mr-tz** [comment]
The file probably contains file.exe's static strings. The error indicates a problem during function emulation.
Can you share the sample or its hash so we can debug this?

**Comment by MavenClaw** [comment]
I'm having the same issue using the Windows x64 prepackaged executable on a Windows 7 platform.  The warnings are shown below.  I was trying to analyze this:  https://virustotal.com/en/file/7c398cea69f3bc305fe5efe20329b81b6803883dfdc9be8aa466517bf091e65a/analysis/

WARNING:vtrace.platforms.win32:LoadLibrary C:\Users\USERNA~1\AppData\Local\Temp\
_MEI21~1\vtrace\platforms\windll\amd64\symsrv.dll: [Error 126] The specified mod
ule could not be found
WARNING:vtrace.platforms.win32:LoadLibrary C:\Users\USERNA~1\AppData\Local\Temp\
_MEI21~1\vtrace\platforms\windll\amd64\dbghelp.dll: [Error 126] The specified mo
dule could not be found

**Comment by williballenthin** [comment]
@MavenClaw the log messages you've included above are warning messages from vivisect. They are non-fatal, so they should not affect the quality of the analysis. If they are a nuisance, you can redirect stderr, since log messages are written to stderr, while final program output is written to stdout.

**Comment by williballenthin** [comment]
should be fixed in 3980f9f0a76e87b91f1f4211057c7fac0a045430
